serveradmin/common/static/js/plugins/esc_dismissible.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Dismiss by ESC keyboard shortcut, Copyright (c) 2021 InnoGames GmbH
+ *
+ * Allows to dismiss (delete) HTML elements that have the class dismissible
+ * by pressing escape.
+ */
+$(document).ready(function() {
+    $(document).keydown(function (event) {
+        // ESC
+        if (event.key === "Escape") {
+            let dismissible = $('.dismissible');
+
+            if (dismissible.length < 1) {
+                return;
+            }
+
+            // Pressing ESC on elements such as inputs makes them loose focus.
+            // To avoid this we remember the focused element to restore it.
+            let input = document.activeElement;
+
+            dismissible.remove();
+
+            if (input) {
+                input.focus();
+            }
+
+            event.preventDefault();
+            event.stopPropagation();
+            return false;
```

Comment:
Could be interesting to know if really both is needed or actually just one of it.

Reply:
Indeed the `return false` seems to be redundant. The MDN Web Docs don't mention `return false` on the pages of `preventDefault` or `stopPropagation` as alternative to achieve the same so I prefer them over the `return false` and remove it.